setup.py

```diff
@@ -4,93 +4,52 @@
 from pathlib import Path
 from shutil import copy
 
-from setuptools import find_packages, setup
+from setuptools import setup
 
 root = Path(__file__).parent
 
-with open(root / "README.md") as f:
-    long_description = f.read()
-with open(root / "requirements-minimal.txt") as f:
-    install_requires = f.readlines()
-with open(root / "requirements-rtd.txt") as f:
-    documentation_dependencies = f.readlines()
-with open(root / "requirements-testing.txt") as f:
-    testing_suite_dependencies = f.readlines()
 
-extras_require = defaultdict(list)
+def read_requirements(file):
+    with open(root / file) as f:
+        return f.readlines()
+
 
-extras_require["dandi"].append("dandi>=0.58.1")
-extras_require["full"].extend(extras_require["dandi"])
+extras_require = defaultdict(list)
+extras_require["full"] = ["dandi>=0.58.1", "hdf5plugin"]
```

Comment:
Hmm so we have to duplicate these extras here and in the `.toml`?

Reply:
Yes, for the moment. I think there is a way of reducing this but this is a big change already and I would like to do it step-wise.